Dockerfile

```diff
@@ -0,0 +1,14 @@
+# This file is part of Flask-RateLimiter
+# Copyright (C) 2015 CERN.
+#
+# Flask-RateLimiter is free software; you can redistribute it and/or
+# modify it under the terms of the Revised BSD License; see LICENSE
+# file for more details.
+
+FROM python:2.7
+ADD . /code
+WORKDIR /code
+RUN pip install pep257
+RUN pip install sphinx
+RUN pip install redis
+RUN python setup.py develop
```

Comment:
> What about using pip for everything: pip install -e .?

This doesn't install `redis` or `pep257`.  We'd have to amend `setup.py` feature list.
Could use `-e .[docs]` though in order to install `sphinx` and friends...